spec/integration/type/sshkey_spec.rb

```diff
@@ -1,22 +0,0 @@
-#! /usr/bin/env ruby
```

Comment:
good question.  i don't even remember taking this out.

looks like 525 tests call ruby out of 752 spec tests.
rspec obviously calls the ruby interpreter.